File: backend/KvantasServer/KvantasServer/Models/Product.cs
namespace KvantasServer.Models
{
    public class Product
    {
        public string Name { get; set; }

        public string Category { get; set; }

        public int Amount { get; set; }

        public int Price { get; set; }

        public override string ToString()
        {
            return $"{{ {nameof(Name)}: \"{Name}\", " +
                $"{nameof(Category)}: \"{Category}\", " +
                $"{nameof(Amount)}: \"{Amount}\", " +
                $"{nameof(Price)}: {Price} }}";
        }
    }
}

File: frontend/nbp-projekat2/src/app/models/Product.ts
export class Product {
    name: string;
    category: string;
    price: number;
    amount :number;
    owner: string;
    username: string;
    constructor(name :string, category :string, price: number ,amount :number, owner: string, username: string) {
        this.name = name;
        this.category = category;
        this.price = price;
        this.amount = amount;
        this.owner = owner;
        this.username = username;
    }
}